### PR TITLE
added community page

### LINKED ISSUE #793: [NG] Add community page
Create the [community page ](https://layer5.io/community) for `layer5-ng` site

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by codetheorem**
Signed-off-by: Hrishikesh Agarwal <hrishikeshagarwalv@gmail.com>

**Description**
Added community page.

This PR fixes #793 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by codetheorem** [comment]
Screen Shot of the Page-

screencapture-localhost-8000-community-1602930267713

File: src/sections/Community/Community.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    opportunities, and more. Our shared commitment to the open source spirit
+    pushes Layer5 projects forward. 
+                </p>
+                <div width="100%" className="center">
+                    <p>New members are always welcome.</p>
+                    <h3>
+                        <a
+                            className=" btn "
+                            href="http://slack.layer5.io"
+                        >Join the Community</a>
```
Comment (Tanuj22) [inline src/sections/Community/Community.js:29]: Use the `Button` component and remove the classes defined for this

File: src/sections/Community/Community.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                <h6>Open Source Projects</h6>
+                <p className="indent">
+    Layer5 projects are open source software. Anyone can download, use, work on,
+    and share it with others. It's built on principles like collaboration,
+    globalism, and innovation. Layer5 projects are distributed under the terms
+    of Apache v2.
+                </p>
+            </div>
+            <div width="100%">
+                <h3 className="h3link">
```
Comment (Tanuj22) [inline src/sections/Community/Community.js:42]: Do not add custom styling for links. Color will be changed later globally for links

File: src/sections/Community/community.style.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    row-view {
+        padding: 15px;
+        vertical-align: middle;
+        height: fit-content;
+    }
+    .center{
+        justify-content:center;
+        text-align: center;
+    }
+    h3 {
```
Comment (Tanuj22) [inline src/sections/Community/community.style.js:20]: Don't change the styles of already defined tags.

File: src/sections/Community/community.style.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        font-size: 2rem;
+        line-height: 110%;
+        margin: 1.46rem 0 1.168rem 0;
+    }
+    h6 {
```
Comment (Tanuj22) [inline src/sections/Community/community.style.js:25]: Don't change the styles of already defined tags.

File: src/sections/Community/community.style.js [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        font-size: 1rem;
+        line-height: 110%;
+        margin: 0.5rem 0 0.4rem 0;
+        font-weight: 500;
+    }
+    h1, h2, h3, h4, h5, h6 {
```
Comment (Tanuj22) [inline src/sections/Community/community.style.js:31]: Don't change the styles of already defined tags.

File: src/sections/Community/community.style.js [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+        font-weight: 400;
+        line-height: 1.1;
+    }
+    a {
```
Comment (Tanuj22) [inline src/sections/Community/community.style.js:35]: Don't change the styles of already defined tags.

File: src/sections/Community/community.style.js [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+        color: #039be5;
+        text-decoration: none;
+        width: auto;
+    }
+    .h3link{
```
Comment (Tanuj22) [inline src/sections/Community/community.style.js:40]: Don't change the styles of already defined tags.

File: src/sections/Community/community.style.js [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+       padding-left:25%;
+    }
+    .link{
+        font-size: 0.65em;
+    }
+    .mentorimg{
+        margin: 5px;
+    }
+    .btn{
```
Comment (Tanuj22) [inline src/sections/Community/community.style.js:49]: remove after using `Button` component

File: src/sections/Community/community.style.js [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+        height: 36px;
+        line-height: 36px;
+        padding: 0 2rem;
+        padding-bottom: 40px;
+        padding-top: 10px;
+        text-transform: uppercase;
+        transition: 0.3s ease-out;
+        box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.14), 0 1px 10px 0 rgba(0, 0, 0, 0.12), 0 2px 4px -1px rgba(0, 0, 0, 0.3);
+    }
+    .btn:hover{
```
Comment (Tanuj22) [inline src/sections/Community/community.style.js:66]: remove

File: src/sections/Community/Community.js [hunk 10]
```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import image from "../../assets/images/community/datacenter-crash.svg";
```
Comment (Tanuj22) [inline src/sections/Community/Community.js:2]: Use a better name than `image`

File: src/sections/Community/CommunityMembers.js [hunk 11]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import image from "../../assets/images/community/Layer5-mentor-program.png";
```
Comment (Tanuj22) [inline src/sections/Community/CommunityMembers.js:2]: same

File: src/sections/Community/community.style.js [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+    }
+    a {
+        color: #039be5;
+        text-decoration: none;
+        width: auto;
+    }
+    .h3link{
+       padding-left:25%;
+    }
+    .link{
```
Comment (Tanuj22) [inline src/sections/Community/community.style.js:43]: remove

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

**Comment by codetheorem** [comment]
@Tanuj22 please review it.

File: src/sections/Community/Community.js [hunk 13]
```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import datacenterimg from "../../assets/images/community/datacenter-crash.svg";
+import Button from "../../reusecore/Button";
+
+const Community= () => {
+    return (
+        <div>
+            <h3>Largest collection of
```
Comment (Tanuj22) [inline src/sections/Community/Community.js:8]: Center this and use h2

File: src/sections/Community/Community.js [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+                <div width="100%" className="center">
+                    <p>New members are always welcome.</p>
+                    <h3>
+                        <a
+                            href="http://slack.layer5.io"
+                        ><Button primary>Join the Community</Button></a>
+                    </h3>
+                </div>
+    
+                <h6>Open Source Projects</h6>
```
Comment (Tanuj22) [inline src/sections/Community/Community.js:33]: use h4

File: src/sections/Community/Community.js [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+    globalism, and innovation. Layer5 projects are distributed under the terms
+    of Apache v2.
+                </p>
+            </div>
+            <div width="100%">
+                <h3>
+                    <a
+                        href="https://docs.google.com/document/d/1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk/edit?usp=sharing"
+                        className="link"
+                    >Repository Overview</a>
```
Comment (Tanuj22) [inline src/sections/Community/Community.js:46]: Center this like the one in current page

File: src/sections/Community/CommunityMembers.js [hunk 16]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import mentorimg from "../../assets/images/community/Layer5-mentor-program.png";
+
+const CommunityMembers=()=>{
+    return (
+        <div>
+            <h3 className="black-text">Our Community Members</h3>
```
Comment (Tanuj22) [inline src/sections/Community/CommunityMembers.js:7]: h2

File: src/sections/Community/CommunityMembers.js [hunk 17]
```diff
@@ -0,0 +1,29 @@
...
+            <img
+                src={mentorimg}
+                width="40%" align="right" 
+                className="mentorimg"
+            />
+            <div>
+                <h6>Jump in! The community is warm</h6>
```
Comment (Tanuj22) [inline src/sections/Community/CommunityMembers.js:14]: h5

File: src/sections/Community/CommunityMembers.js [hunk 18]
```diff
@@ -0,0 +1,29 @@
...
+  The Layer5 community is diverse with members from around the globe, who have shown consistent commitment in advancing our open source initiatives.
+                <p className="indent">
+  We believe that every contributor deserves a safe and friendly environment, enabling them to effectively compare different ideas and find the best solutions for advancement, while building the size, diversity, and strength of the community.
+                </p>    
+                <div width="100%" className="center">
+                    <h3>
+                        <a href="/community/members">See our community members</a>
```
Comment (Tanuj22) [inline src/sections/Community/CommunityMembers.js:21]: use `Link`

File: src/sections/Community/MeshmatesProgramSummary.js [hunk 19]
```diff
@@ -0,0 +1,28 @@
...
+                <p className="indent">
+    The Layer5 community is growing at a tremendous rate. We value our connections and that is what makes us unique. Whether it is your first Layer5 meeting or your tenth contribution, we want to make sure that you are supported and equipped to take advantage of all of that the Layer5 community has to offer. 
+                </p>
+                <p className="indent">  
+    In the Layer5 mentor program, we will match you up with a MeshMate to support and guide you along the way, by enabling you to identify your area of interests within the projects, directing you towards working groups to join, growing your Cloud Native knowledge, and establishing a network of relationships.
+                </p>
+                <div width="100%" className="center">
+                    <h3>
+                        <a
+                            href="/community/meshmates"
```
Comment (Tanuj22) [inline src/sections/Community/MeshmatesProgramSummary.js:20]: use `Link`

File: src/sections/Community/MeshmatesProgramSummary.js [hunk 20]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import meshmateimg from "../../assets/images/community/meshmate-stack.svg"
+ 
+export default function MeshmatesProgramSummary() {
+    return (
+        <div>
+            <div>
+                <h3>Layer5 MeshMates</h3>
```
Comment (Tanuj22) [inline src/sections/Community/MeshmatesProgramSummary.js:8]: h2

File: src/sections/Community/MeshmatesProgramSummary.js [hunk 21]
```diff
@@ -0,0 +1,28 @@
...
+                <img src={meshmateimg} align="right" width="40%" />
+                <h6>An onboarding and mentoring program</h6>
```
Comment (Tanuj22) [inline src/sections/Community/MeshmatesProgramSummary.js:10]: h5

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Also, the table is not responsive in mobile view

**Comment by codetheorem** [comment]
I have added overflow-x auto so that anybody can scroll in horizontal to view table.

**Review by Tanuj22 — APPROVED** [review]
Looks good. Thanks for working on this :)

**Pull request merged**